core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -184,6 +186,7 @@ contract Liquidatable is PricelessPositionManager {
     function createLiquidation(address sponsor) public returns (uint uuid) {
         // Attempt to retrieve Position data for sponsor
         PositionData storage positionToLiquidate = _getPositionData(sponsor);
+        require(positionToLiquidate.collateral.rawValue > 0, "Cant liquidate a position with no collateral");
```

Comment:
why not use `FixedPoint.isGreaterThan(positionToLiquidate.collateral, 0)`?

Reply:
ye, that's better. updated